internal/constants/constants.go

```diff
@@ -9,7 +9,7 @@ const (
 
 	// Set the prefix that needs to be used when setting the configuration
 	// using environment variables
-	EnvVarPrefix = "amidostacks"
+	EnvVarPrefix = "ensonostacks"
```

Comment:
Added ability to set another enviornment variable that is read out of band. This can have the name of the prefix which will change how the CLI looks at environment vaariables.
Have updated the documentation to show this functionaility.

Reply:
Noice